files/Dockerfile

```diff
@@ -18,7 +18,12 @@ RUN docker-php-source extract
 # string (up the the !) is the pattern of the lines that will be ignored.
 # Needed for apt-get build-dep call later in script
 #
-RUN sed -Ei '/.*partner/! s/^# (deb-src .*)/\1/g' /etc/apt/sources.list
+
+# PHP 8.0
```

Comment:
Can we automatically toggle between the two depending on the value of PHP_VER?

Reply:
We could, but having investigated that before I decided it would make too much of a mess here and opted for the simpler solution